src/client/stylesheets/card.css

```diff
@@ -0,0 +1,4 @@
+.card-text{
```

Comment:
white text with black boarder shows well on any background color, so all text on the cards will do this.  The text is a little bit more blocky so let me know what you guys think of this.